src/main/java/com/avispl/symphony/dal/communicator/lumen/vc/tr60a/LumenVCTR60ACommunicator.java

```diff
@@ -1819,12 +1820,20 @@ protected byte[] read(byte[] command) throws IOException {
 		byte[] response;
 
 		do {
-			response = super.read(command);
-			responseString = getHexByteString(response);
+			try {
+				response = super.read(command);
+				responseString = getHexByteString(response);
 
-			// If this is ACK packet, need to save to the flag to check done reading
-			if (responseString.endsWith(getHexByteString(ReplyStatus.ACK.getCode()))) {
-				hasACK = true;
+				// If this is ACK packet, need to save to the flag to check done reading
+				if (responseString.endsWith(getHexByteString(ReplyStatus.ACK.getCode()))) {
+					hasACK = true;
+				}
+			} catch (IOException e) {
+				logger.error("Error reading response from socket, socket server might close", e);
```

Comment:
I think it's ok to just have an exception with a clear message, having error log + exception may be a bit redundant